repos/fdbt-netex-output/src/types/index.ts

```diff
@@ -278,7 +278,7 @@ export const isMultiServiceTicket = (ticket: Ticket): ticket is PeriodMultipleSe
     (ticket as PeriodMultipleServicesTicket).selectedServices !== undefined;
 
 export const isPeriodGeoZoneTicket = (ticket: Ticket): ticket is PeriodGeoZoneTicket =>
-    ticket.type === 'period' && (ticket as PeriodGeoZoneTicket).zoneName !== undefined;
+    !isSchemeOperatorTicket(ticket) && ticket.type === 'period' && (ticket as PeriodGeoZoneTicket).zoneName !== undefined;
```

Comment:
Minor: is it worth changing this to a proper type guard?
```
ticket.type === 'period' && 'zoneName' in ticket;
```

Reply:
Yep good shout - will do